recipes/lalinference/meta.yaml

```diff
@@ -0,0 +1,189 @@
+{% set name = "lalinference" %}
+{% set version = "1.10.0" %}
+{% set sha256 = "e44af2856d9570aeb795be3a663a7dd7bc6e65fa07f46846da4b0fd2c0537fa5" %}
+
+package:
+  name: {{ name }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.xz
+  url: http://software.ligo.org/lscsoft/source/lalsuite/{{ name }}-{{ version }}.tar.xz
+  sha256: {{ sha256 }}
+  patches:
+    # fix python3 syntax
+    - lalinference-python3.patch
+
+build:
+  number: 0
+  skip: true  # [win]
+
+requirements:
+  build:
+    - pkg-config
+    - make
+    - {{ compiler('c') }}
+  host:
+    - swig >=3.0.7
+    - gsl
+    - metaio
+    - chealpix >=3.30.0
+    - lal >=6.19.0
+    - lalmetaio >=1.4.0
+    - lalsimulation >=1.8.0
+    - lalburst >=1.5.0
+    - lalinspiral >=1.8.0
+    - lalpulsar >=1.17.0
+
+outputs:
+  - name: lalinference
+    script: install-c.sh
+    requirements:
+      host:
+        - {{ compiler('c') }}
+        - make
```

Comment:
compiler and make probably don't need to be host dependencies, just build deps.

Reply:
@scopatz, whenever I try that, my environment in the `install-interface.sh` script doesn't pick up the `build` environment, just the `host` one, so I shoved everything into `host`. Is that expected?

Reply:
@scopatz, did you find out anything more about this?

Reply:
no sorry, I haven't looked into it further

Reply:
other than making the build more complicated than it could be, is there a downside of my workaround that will impact the built package? If not, can we merge now anyway? I would be happy to try and reproduce this problem with a simpler setup and post an issue on conda/conda-build, or similar.

Reply:
@conda-forge/staged-recipes, can we merge this anyway (see note above)?